Altis_Life.Altis/core/functions/fn_escInterupt.sqf

```diff
@@ -74,5 +74,8 @@ for "_i" from 0 to 1 step 0 do {
     if (_usebleCtrl) then {
         _respawnButton ctrlEnable true; //Enable the button.
     };
-    waitUntil {isNull (findDisplay 49)};
+    waitUntil {isNull (findDisplay 49) || {!alive player}};
+    if (!isNull (findDisplay 49)) then {
```

Comment:
Although unlikely, they could *possibly* reopen the menu after closing it (while alive), causing it to be closed again. Worth adding another dead check in the !isNull if statement. Will commit, for you.